Library/Homebrew/utils/github.rb

```diff
@@ -309,8 +309,8 @@ def repository(user, repo)
     open_api(url_to("repos", user, repo))
   end
 
-  def search_code(**qualifiers)
-    matches = search("code", **qualifiers)
+  def search_code(user: "Homebrew", path: ["Formula", "Casks", "."], filename: nil, extension: "rb")
```

Comment:
I think these defaults except `filename: nil` are better kept in the `search.rb`